gui/wxpython/gui_core/goutput.py

```diff
@@ -620,7 +620,7 @@ def AddStyledMessage(self, message, style=None):
                     self.linePos -= 1
                 else:
                     if c == "\r":
-                        pos = self.GetCurLine()[1]
```

Comment:
this can be removed

Reply:
Now it technically changes the behavior because you removed the if, but I am ok with simplifying it.